package.json

```diff
@@ -18,17 +18,17 @@
     "babel-eslint": "10.0.1",
     "babel-loader": "8.0.4",
     "del": "2.0.2",
-    "eslint": "5.7.0",
-    "eslint-loader": "2.1.1",
-    "gulp": "4.0.0",
+    "eslint": "7.0.0",
```

Comment:
there are newer versions of eslint, but they are not compatible with eslint-loader....